ambari-web/app/styles/bootstrap_overrides.less

```diff
@@ -506,6 +506,7 @@ select.form-control {
   background-color: inherit;
 }
 .wizard .wizard-body .wizard-nav {
+  min-width: 250px;
```

Comment:
On narrow screen wizard-body will jump to bottom, better to leave previous styles

Reply:
No, my changes ensure that the wizard-body stays where it should even if the screen is very narrow (although hopefully nobody is using Ambari on that small of a screen, since the UI is really not designed for it).

![image](https://user-images.githubusercontent.com/13603638/35448783-3dbe4490-0289-11e8-85f0-482a18cfb1c7.png)